spec/client_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe 'rsyslog::client' do
   cached(:chef_run) do
-    ChefSpec::ServerRunner.new(platform: 'ubuntu', version: '16.04') do |node|
+    ChefSpec::ServerRunner.new(platform: 'ubuntu', version: '20.04') do |node|
```

Comment:
You can save yourself some pain and just remove the version here so you don't need to update it again